share/empty.go

```diff
@@ -13,6 +14,29 @@ import (
 	"github.com/celestiaorg/rsmt2d"
 )
 
+var (
+	emptyRoot atomic.Pointer[Root]
+	emptyEDS  atomic.Pointer[rsmt2d.ExtendedDataSquare]
```

Comment:
Why do you need atomic if you only write during init?

Reply:
True, originally, I did that lazily with atomict, but now it makes sense to avoid atomic at all